searchcore/src/vespa/searchcore/proton/documentmetastore/documentmetastoreinitializer.cpp

```diff
@@ -49,7 +49,7 @@ DocumentMetaStoreInitializer::run()
             if (!_dms->load()) {
                 throw IllegalStateException(failedMsg(_docTypeName.c_str()));
             } else {
-                _dms->commit(snap.syncToken, snap.syncToken);
+                _dms->commit(search::CommitParam(snap.syncToken, true));
```

Comment:
Why are we using _forceUpdateStats=true_ here? That is not equal to the previous behaviour.